src/applications/appeals/996/tests/hlr-keyboard-only.cypress.spec.js

```diff
@@ -35,12 +37,12 @@ describe('Higher-Level Review keyboard only navigation', () => {
       );
       cy.injectAxeThenAxeCheck();
 
-      // Wizard
+      // Subtask
       cy.url().should('include', '/start');
-      cy.tabToElement('input[value="compensation"]');
+      cy.tabToElement('input#compensationinput'); // ID of va-radio-option input
       cy.realPress('Space');
 
-      cy.tabToElement('.vads-c-action-link--green');
+      cy.tabToElement('va-button');
```

Comment:
Could you add the containing class to this selector?

Reply:
Done